src/Symfony/Component/VarDumper/Caster/ConstStub.php

```diff
@@ -30,4 +30,27 @@ public function __toString()
     {
         return (string) $this->value;
     }
+
+    /**
+     * Creates ConstStub from bitfield flag.
+     *
+     * @param int    $value  The value of bitfield flag
+     * @param string $prefix The prefix filter applied on constant names
+     *
+     * @return self
+     */
+    public static function fromFlag($value, $prefix)
+    {
+        $constants = get_defined_constants();
+        foreach ($constants as $c => $v) {
+            if ('' !== $prefix && 0 !== strpos($c, $prefix) || ($value & $v) !== $v) {
```

Comment:
For a wider usage (with class constants), maybe you could exclude non integer values (otherwise you'll may get a `PHP warning: A non-numeric value encountered`).
Also maybe only keep single bits (by using something like `0 === ($v & $v - 1)` where `$v = 0b00010` will return `true`, but `$v = 0b00011` will return `false`), so we can't have combinations (`E_ALL`) + singles bits nor non-related constants.